pybotvac/robot.py

```diff
@@ -56,11 +56,16 @@ def _message(self, json):
         response.raise_for_status()
         return response
 
-    def start_cleaning(self, mode=2, navigation_mode=2):
+    def start_cleaning(self, mode=2, navigation_mode=2, category=None):
         # mode & naivigation_mode used if applicable to service version
         # mode: 1 eco, 2 turbo
         # navigation_mode: 1 normal, 2 extra care
-        
+        # category: 2 non-persistent map, 4 persistent map
+
+        #Default to using the persistent map if we support basic-3.
+        if category is None:
+            category = 3 if self.service_version == 'basic-3' else 2
```

Comment:
I think this should be a `4` instead of a `3` because `3` is used for spot cleaning only.

Reply:
Good catch, that should be a 4.